tests/e2e/features/actions/mainPage/faucet/actions-faucet.feature

```diff
@@ -0,0 +1,15 @@
+@action @smoke @regression @faucet @emptyWallet
+Feature: Faucet
+
+  Background:
+    Given Connect Metamask extension with login action
+
+  @id1550
+  Scenario: Check Faucet available only on Testnet
```

Comment:
I think it will be more easy to call it just "Check Faucet NOT available on Mainnet" and add a minnet tag to this test

Reply:
Fixed in latest commit